apps/mosaico/src/Main.purs

```diff
@@ -224,6 +232,33 @@ frontpage { htmlTemplate } _ = do
             appendMosaico mosaicoString htmlTemplate >>= appendHead windowVars
   pure $ TextHtml html
 
+tagList :: Env -> { params :: { tag :: String }, guards :: { credentials :: Maybe UserAuth } } -> Aff TextHtml
+tagList { htmlTemplate } { params: { tag } } = do
+  articles <- Lettera.getByTag 0 10 (uriComponentToTag tag) HBL
+  mostReadArticles <- Lettera.getMostRead 0 10 "" HBL true
+  mosaico <- liftEffect MosaicoServer.app
+  frontpageComponent <- liftEffect Frontpage.frontpageComponent
+  let mosaicoString =
+        DOM.renderToString
+        $ mosaico
+          { mainContent:
+              TagListContent
+              $ frontpageComponent
+                  { frontpageArticles: articles
+                  , onArticleClick: const $ pure unit
+                  , onTagClick: const $ pure unit
+                  }
+          , mostReadArticles
+          }
+  html <- liftEffect do
+            let windowVars =
+                  "<script>\
+                     \window.tagListArticles=" <> encodeStringifyArticleStubs articles <> ";\
```

Comment:
In addition to writing this to the template, we should also set it as a prop to Mosaico so it doesn't fetch them again when the browser routes kick in.

So changes are needed to the `Mosaico.purs` module and this javascript file, where we actually start the thing: https://github.com/KSF-Media/affresco/blob/master/apps/mosaico/web/index.js#L14-L20

Although, as I am working with the category pages currently, we will have a lot of different named feeds, which are all basically the same thing, so unifying them will probably be a good idea (in another pull request though, maybe in the one I'm working on). In any case, let's add the `tagListArticles` as a prop for now.

Reply:
I added `tagListArticles` to `web/index.js` as well.